docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -406,6 +406,18 @@ This document refers to the `http://localhost:8080/q/dev-ui` Dev UI URL in sever
 If you customize `quarkus.http.root-path` or `quarkus.http.non-application-root-path` properties, then replace `q` accordingly.
 For more information, see the https://quarkus.io/blog/path-resolution-in-quarkus/[Path resolution in Quarkus] blog post.
 
+== OIDC Lightweight Dev Services
+
+If you require advanced setup, the Keycloak Dev Services are perfect choice.
```

Comment:
It is not really about having an advanced startup, but about Keycloak users preferring to test against the real provider, against real world realms that have to be imported in devmode.

Reply:
I literally copied `advanced setup` it from https://github.com/quarkusio/quarkus/issues/31340 where you wrote `some more advanced realm setup` :-) Nevertheless, I think your suggestion is good an I will fix it. Thanks

Reply:
I re-read https://github.com/quarkusio/quarkus/issues/31340 and I think it tries to suggest that this feature is mainly for non-Keycloak users working in dev mode. Must admit the speed is main advantage in my eyes.

Reply:
Sorry if this sounded too defensive, I just didn't realize difference in our views. I can understand why someone wants to use "a real provider", but as someone who spends most of his live just writing tests, all I want is to have them run quickly, no waiting :-) I can't really leverage starting one container a day and use it all day.

Reply:
@michalvavrik Np at all, it is just not fair IMHO toward Keycloak users who now depend on the live container starting having to start doing something extra because another group of users are more interested in the start up time savings as opposed to getting anything from the actual live container.

Reply:
I rewrote docs additional entirely, please have a look.